app/web/src/newhotness/layout_components/AttributeInput.vue

```diff
@@ -1052,11 +1031,6 @@ const selectedConnection = computed(
 //     fixWindowPosition();
 //   });
 // });
-
-// TODO(Wendy) - add the ability to clear a value!
-// const clearValue = (e: Event) => {
-//   e.preventDefault();
-// };
```

Comment:
We can re-introduce if need be, but it remains unused for now and is not needed for both remove subscription scenarios.

Source: https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault